EDA.Rmd

```diff
@@ -1,595 +1,596 @@
+Los modelos son una herramienta para extraer los patrones que existen en los datos. Por ejemplo, considera los datos sobre los diamentes. Resulta difícil entender la relación entre corte y precio, pues corte y quilate, así como quilate y precio están estrechamente relacionadas. Es posible usar un modelo para remover la fuerte relación entre precio y quilate de manera que podamos explorar las sutilezas que quedan en los datos. El código mostrado a continuación crea un modelo que predice el `precio` a partir de la variable `quilate` y después calcula los residuales (la diferencia entre la variable predecida y el valor real). Los residuales nos informan acerca del precio de un diamante, una vez que el efecto que el quilataje tiene sobre esta variable ha sido removido.
 
 ```{r, dev = "png"}
 library(modelr)
 
-mod <- lm(log(price) ~ log(carat), data = diamonds)
+mod <- lm(log(precio) ~ log(quilate), data = diamantes)
 
-diamonds2 <- diamonds %>%
-  add_residuals(mod) %>%
+diamantes2 <- diamantes %>% 
+  add_residuals(mod) %>% 
   mutate(resid = exp(resid))
 
-ggplot(data = diamonds2) +
-  geom_point(mapping = aes(x = carat, y = resid))
+ggplot(data = diamantes2) + 
+  geom_point(mapping = aes(x = quilate, y = resid))
 ```
 
-Once you've removed the strong relationship between carat and price, you can see what you expect in the relationship between cut and price: relative to their size, better quality diamonds are more expensive. 
+Una vez que has removido la relación entre quilate y precio, puedes observar lo que esperarías sobre la relación entre corte y precio: los diamantes de mejor calidad son más costosos en relación con su tamaño.
 
 ```{r}
-ggplot(data = diamonds2) +
-  geom_boxplot(mapping = aes(x = cut, y = resid))
+ggplot(data = diamantes2) + 
+  geom_boxplot(mapping = aes(x = corte, y = resid))
 ```
 
-You'll learn how models, and the modelr package, work in the final part of the book, [model](#model-intro). We're saving modelling for later because understanding what models are and how they work is easiest once you have tools of data wrangling and programming in hand.
+En la parte final de este libro, [model](#model-intro), aprenderás cómo funciona el modelado estadístico y el paquete **modelr**. Dejamos el modelado para más adelante pues es más fácil entender qué son los modelos y cómo funcionan una vez que cuentes con las herramientas necesarias de doma de datos y programación.
 
-## ggplot2 calls
+## Argumentos en ggplot2
 
-As we move on from these introductory chapters, we'll transition to a more concise expression of ggplot2 code. So far we've been very explicit, which is helpful when you are learning:
+A medida que nos alejamos de estos capítulos introductorios, usaremos expresiones más concisas para escribir código ggplot2. Hasta ahora hemos sido muy explícitos en el código, lo que es muy útil cuando estás aprendiendo:
 
 ```{r, eval = FALSE}
```

Comment:
Hola, acabo de agregar el data set `fiel` dentro del paquete `datos`: 

``` r
head(datos::fiel)
#>   erupciones espera
#> 1      3.600     79
#> 2      1.800     54
#> 3      3.333     74
#> 4      2.283     62
#> 5      4.533     85
#> 6      2.883     55
```

<sup>Created on 2018-12-19 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

Reply:
@edgararuiz ¿Ud. lo agregó en edgarruiz/datos o en cienciadedatos/datos?

Reply:
`cienciadedatos/datos`